### ISSUE TITLE
pressflow6 version from drupal6.49

### ISSUE THREAD

**Issue by timlie** [pressflow/6#127]
Hi,

Is it possible to release new version of pressflow6 based of drupal6.49?
This version has better PHP 7.2 support.

Thanks!

**Comment by pwolanin** [comment]
yes, have gotten behind on this but looking into getting up to date

**Comment by timlie** [comment]
Thanks!

**Comment by jvieille** [comment]
Shall we consider this release the latest Pressflow ?
Apparently, beside some mess in packaging info and version numbers, there is only a small change "Fix Drush issues from SA-CORE-2019-002" in 6.48 + 6.49  D6lts release

Thanks
